raiden/transfer/token_network.py

```diff
@@ -122,6 +132,53 @@ def handle_settled(
     )
 
 
+def handle_batch_unlock(
+        token_network_state,
```

Comment:
also general comment: For new functions try to be adding types. Especially since you moved this state transition function from [transfer/node.py](https://github.com/raiden-network/raiden/pull/2020/files#diff-48b12365b69240606c9666e50f5aac2fL556) where it had types.